### ISSUE TITLE
Re-bump the repositories#show page to show more information

### ISSUE THREAD

**Issue by mssola** [SUSE/Portus#749]
## The repository page

We should be showing some statistics in the repo page. Some info that could
be added:
- Repository pulls & pushes.
- Contributors.
- Stars (we are already doing it, but we might consider moving them).
- Description (the same as for stars).

We should also show more information for each tag:
- Size of each tag.
- Image ID (already reported).
- Download button that copies to the clipboard how to pull the tag.

Last but not least, it would be awesome if we could have a timeline for a
specific repository.
## Visualize

We should add a way to visualize all the information from the layers that exist
on the image. See #200 and #249.
## Open Questions

There are some open questions that should be addressed before starting to work
on this:
- How do we allow permissions per repository ? (See #517)
- How do we prevent overwrite of tags ? (See #627)
## Tasks
- [ ] Add statistics
- [ ] Show more information for each tag
- [ ] Activities list for the specific repository.
- [ ] Implement the visualization of the repository.

**Comment by mssola** [comment]
There's some preliminary work on clipboard support in the issue #697.

**Comment by hbjcr** [comment]
Is this feature going to be included in your next milestone? This would help us enable automated builds on our end.

**Comment by mssola** [comment]
@hbjcr we are still planning what to do for the next release. I'd personally like to have it included, but I cannot give you any guarantees.

**Comment by vitoravelino** [comment]
Closing this in favor of #2074, #2073, #2072 and #2071.
